jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -208,6 +208,23 @@ def test_qr(self, harness: primitive_harness.Harness):
       with self.assertRaisesRegex(expected_error, "Unsupported dtype"):
         harness.dyn_fun(*harness.dyn_args_maker(self.rng()))
 
+  @primitive_harness.parameterized(primitive_harness.lax_linalg_svd)
+  def test_svd(self, harness: primitive_harness.Harness):
+    if harness.params["dtype"] in [jnp.complex64, jnp.complex128]:
+      if jtu.device_under_test() == "tpu":
+        raise unittest.SkipTest("No TPU implementation for SVD of complex numbers")
```

Comment:
@gnecula not exactly sure what this would throw on TPU, I assume it'd be a `RuntimeError`? I will need your help to make that into an appropriate `assertRaisesRegex`.